wandb/util.py

```diff
@@ -1247,3 +1249,23 @@ def _is_databricks():
                 sc = shell.sc
                 return sc.appName == "Databricks Shell"
     return False
+
+
+def handle_sweep_config_violations(warnings):
+    """Render warnings from gorilla describing the ways in which a 
+    sweep config violates the allowed schema as terminal warnings.
+
+    Parameters
+    ----------
+    warnings: list of str
+        The warnings to render.
+    """
+    warning_base = (
+        "Malformed sweep config detected! This may cause your sweep to behave in unexpected ways. "
+        "To avoid this, please fix the sweep config schema violations below:\n\n"
+    )
+    for i, warning in enumerate(warnings):
+        warning_base += "Violation {}. {}\n\n".format(i + 1, warning)
+
+    if len(warnings) > 0:
+        term.termwarn(warning_base)
```

Comment:
I'm not a fan of the 2 newlines and WARNING column.  I would either get rid of the newlines, or change it to single newlines and just print regular `term.termlog` messages for the newlines instead of the WARNINGs

Reply:
done